### PR TITLE
chore(@toss/rollup-config): Improve readbility for the rollup settings

### PR DESCRIPTION
**Pull request by po4tion**
## Overview

Due to the nature of "Open source", I think it should provide good readability to people who read code for the first time. So I improved readability through clear allocation of variables and step-by-step processing.

## PR Checklist

- [X] I read and included theses actions below

1. I have read the [Contributing Guide](https://github.com/toss/slash/blob/main/.github/CONTRIBUTING.md)
2. I have written documents and tests, if needed.

### CODE REVIEW

**Review by okinawaa — COMMENTED** [review]
The key value of `packageJSON` has already been assigned as `dependencies` and `peerDependencies`, so the developer seems to have no problem reading it,

I think these changes create unnecessary const

Is there any reason or rationale for this change to improve readability?

**Comment by po4tion** [comment]
Even if this code doesn't change, you'll have no problem using the "slash" library. But if you think about it from the perspective of a developer who reads the code for the first time, it's simple.

Code you don't write yourself is bound to debug. A clear code structure provides convenience for developers trying to debug.

**Comment by okinawaa** [comment]
>Code you don't write yourself is bound to debug. A clear code structure provides convenience for developers trying to debug. 

Im fully agree with your opinion 😄

What I want to know is what point(reason) is the new code is more readable than the old code.
Is there a basis or an indicator for the suggestion

**Comment by po4tion** [comment]
I'd like to apologize first.

I found out that I made an error while talking to you. I thought I'd changed the code for readability so far, but it wasn't.

What I ultimately wanted to pursue in this "PR" is clarity. Old code is concise and readable. But rather than that, I focused on making it clear and easy for developers to read the code.

I think changing the flow of code to be more readable, even if it's a little bit of a brevity, is something that enhances the developer's experience.

If you agree with me, I will change the title of "PR".

I present the data that I refer to. [Link](https://www.freecodecamp.org/news/how-to-write-clean-code/#conciseness-versus-clarity)

To sum up, it is important to balance simplicity and clarity for the developer's experience.

**Review by okinawaa — APPROVED** [review]
LGTM👍

**Comment by okinawaa** [comment]
Thank you for writing the good PR description and writing the comments well🎆
